salsah/src/public/js/jquery.searchbox.js

```diff
@@ -217,6 +224,15 @@
 								$this.css({cursor: 'text'});
 							}
 							localdata.ele.sel.hide();
+							// when leaving this input, if the value is empty, but we had a previous valid input
+							// set back the previous input
+							if ($this.attr("isValid") && $this.attr("isValid") == "false") {
```

Comment:
To be honest: the old GUI is full of JS bugs and this is also my fault (I also blame @lrosenth) ;-)